pym/portage/sync/modules/git/git.py

```diff
@@ -130,3 +130,15 @@ def update(self):
 			cwd=portage._unicode_encode(self.repo.location))
 
 		return (os.EX_OK, current_rev != previous_rev)
+
+	def retrieve_head(self, **kwargs):
+		'''Get information about the head commit'''
+		if kwargs:
+			self._kwargs(kwargs)
```

Comment:
Okay, sounds good to me. I was relating to https://github.com/gentoo/portage/pull/173/files#diff-290c5cc3f4bd746508c90fd4d6e5c052R1654 if options is set up correctly of if there's a different way to do it.